examples/call-service-panel-example/eslint.config.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+
+const foxglove = require("@foxglove/eslint-plugin");
+const globals = require("globals");
+const tseslint = require("typescript-eslint");
```

Comment:
typescript-eslint is missing from package.json (I think this applies to all the examples)

Reply:
🤔 I did run `npm run lint` in each example (installing packages only in that example) and it ran. Maybe something else had it as a transient.

Reply:
https://www.npmjs.com/package/@foxglove/eslint-plugin?activeTab=dependencies It is already a dependency of `@foxglove/eslint-plugin`

Reply:
Sure but don’t we always ensure that we declare dependencies on things we explicitly import? That is what our dependency lint step does in the app repo.

Reply:
🤔 Yea I could see that. I guess my dream is the users worry about fewer dependencies.